src/pages/ReimbursementAccount/ACHContractStep.js

```diff
@@ -100,44 +89,35 @@ class ACHContractStep extends React.Component {
     /**
      * @param {Number} ownerKey - ID connected to the beneficial owner identity form
      */
-    removeBeneficialOwner(ownerKey) {
-        this.setState((prevState) => {
-            const beneficialOwners = _.without(prevState.beneficialOwners, ownerKey);
-
-            FormActions.setDraftValues(ONYXKEYS.REIMBURSEMENT_ACCOUNT, {beneficialOwners});
-
-            return {beneficialOwners};
-        });
+    const removeBeneficialOwner = (ownerKey) => {
+        const previousBeneficialOwners = _.without(beneficialOwners, ownerKey);
+        FormActions.setDraftValues(ONYXKEYS.REIMBURSEMENT_ACCOUNT, {previousBeneficialOwners});
+        setBeneficialOwners(previousBeneficialOwners);
     }
 
-    addBeneficialOwner() {
-        this.setState((prevState) => {
-            // Each beneficial owner is assigned a unique key that will connect it to an Identity Form.
-            // That way we can dynamically render each Identity Form based on which keys are present in the beneficial owners array.
-            const beneficialOwners = [...prevState.beneficialOwners, Str.guid()];
-
-            FormActions.setDraftValues(ONYXKEYS.REIMBURSEMENT_ACCOUNT, {beneficialOwners});
-            return {beneficialOwners};
-        });
+    const addBeneficialOwner = () => {
+        // Each beneficial owner is assigned a unique key that will connect it to an Identity Form.
+        // That way we can dynamically render each Identity Form based on which keys are present in the beneficial owners array.
+        const newBeneficialOwners = [...beneficialOwners, Str.guid()];
+        FormActions.setDraftValues(ONYXKEYS.REIMBURSEMENT_ACCOUNT, {beneficialOwners});
+        setBeneficialOwners(newBeneficialOwners);
     }
 
     /**
      * @param {Boolean} ownsMoreThan25Percent
      * @returns {Boolean}
      */
-    canAddMoreBeneficialOwners(ownsMoreThan25Percent) {
-        return _.size(this.state.beneficialOwners) < 3 || (_.size(this.state.beneficialOwners) === 3 && !ownsMoreThan25Percent);
-    }
+    const canAddMoreBeneficialOwners = (ownsMoreThan25Percent) => _.size(beneficialOwners) < 3 || (_.size(beneficialOwners) === 3 && !ownsMoreThan25Percent);
 
     /**
      * @param {Object} values - object containing form input values
      */
-    submit(values) {
-        const bankAccountID = lodashGet(this.props.reimbursementAccount, 'achData.bankAccountID') || 0;
+    const submit = (values) => {
+        const bankAccountID = lodashGet(props.reimbursementAccount, 'achData.bankAccountID') || 0;
 
-        const beneficialOwners = !values.hasOtherBeneficialOwners
+        const updatedBeneficialOwners = !values.hasOtherBeneficialOwners
```

Comment:
Because variable name `beneficialOwners` is the same with state variable name.
So I thought it might to bring some confusion or potential risk in further development.
There was any special purpose or meaning beside above reason.
If you require to change it, I will do.